build_tools/github_actions/fetch_test_configurations.py

```diff
@@ -181,7 +181,7 @@ def _get_script_path(script_name: str) -> str:
     "rccl": {
         "job_name": "rccl",
         "fetch_artifact_args": "--rccl --tests",
-        "timeout_minutes": 15,
+        "timeout_minutes": 60,
```

Comment:
Same here - 60 minutes was not enough time for the rccl tests: https://github.com/ROCm/TheRock/actions/runs/19711981454/job/56488728144?pr=2285